src/EscapeAnalysis.jl

```diff
@@ -250,11 +259,28 @@ end
 ⋤(x::EscapeLattice, y::EscapeLattice) = !⊑(y, x)
```

Comment:
Unrelated to this PR, I noticed this formula is incomplete, since there isn't a total order anymore:
```
⋤(x::EscapeLattice, y::EscapeLattice) = ⊑(x, y) && !⊑(y, x)
```

Reply:
Ah, true. Thanks for your point.